Library/Formula/terraform-inventory.rb

```diff
@@ -0,0 +1,31 @@
+class TerraformInventory < Formula
+  desc "Terraform State → Ansible Dynamic Inventory"
+  homepage "https://github.com/adammck/terraform-inventory"
+  url "https://github.com/adammck/terraform-inventory/archive/v0.6.tar.gz"
+  sha256 "af09a2606db728fd046e0ce5a6960386e5f766f8d187eea15220764fd669187e"
+  head "https://github.com/adammck/terraform-inventory.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    # Move the contents of the repo (which are currently in the buildpath) into
+    # a go-style subdir, so we can build it without spewing deps everywhere.
+    app_path = buildpath/"src/github.com/adammck/terraform-inventory"
+    app_path.install Dir["*"]
+
+    # Fetch the deps (into our temporary gopath) and build
+    cd "src/github.com/adammck/terraform-inventory" do
+      system "go", "get"
```

Comment:
The dependencies need to be specified as `go_resources` so we can checksum and download them in advance with `brew fetch`; see [`hugo.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/hugo.rb) for an example.